README.md

```diff
@@ -86,6 +86,35 @@ Play-Json extensions
     Json.parse("""{"i": 5, "s":"foo", "type": "X"}""").as[SomeAdt] == X(5,"foo")
 
 ### experimental features (will change)
+
+#### Serialization of polymorphic types with tags
+To overcome ambiguities when using `formatSealed` one option is to use type-tags.
+`formatTagged` adds type-tag field during serialization and relies on this field during deserialization.
+`formatSealedTagged` also inspects type-tag information to choose correct child Format.
+
+    sealed trait ApiRequest
+    case class Withdraw(amount: BigDecimal) extends ApiRequest
+    case class Deposit(amount: BigDecimal) extends ApiRequest
+
+    object ApiRequest {
+      import play.api.libs.json.Json
+      import org.cvogt.play.json.{Jsonx, Tags}
+
+      // Define tagging strategy
+      implicit val tags = Tags.CaseInsensitivePreservingShortTags(field = "_type")
+
+      // Wrap standard formats with tagged ones
+      implicit private val depositFmt = Jsonx.formatTagged(Json.format[Deposit])
+      implicit private val withdrawFmt = Jsonx.formatTagged(Json.format[Withdraw])
+      implicit val apiFmt = Jsonx.formatSealedTagged[ApiRequest]
+    }
+
+    val deposit = Deposit(10)
+    val withdraw = Withdraw(10)
+    assert(Json.parse(s"""{"_type": "withdraw", "amount": 10.0}""").as[ApiRequest] === withdraw)
+    assert(Json.parse(s"""{"_type": "deposit", "amount": 10.0}""").as[ApiRequest] === deposit)
+
+
```

Comment:
But what you think about leaving `formatSealedTagged`, but without necessary depending on `formatTagged` of child formats?